snippets/snippets.json

```
@@ -11,14 +11,14 @@
       "\t\t\tconst commandId = 'vscode.didact.register';",
       "\t\t\tconst tutorialName = `${1:Your First Integration}`; // replace this with the name of your integration to appear in the Didact view",
       "\t\t\tconst tutorialPath = path.resolve(context.extensionPath, `${2:./path/to/tutorial.didact.md|adoc}`); // replace this path with the location of your Didact file like ./path/to/tutorial.didact.md or ./path/to/tutorial.didact.adoc",
-      "\t\t\tconst tutorialUri = vscode.Uri.file(`${tutorialPath\\}`);",
+      "\t\t\tconst tutorialUri = vscode.Uri.file(context.asAbsolutePath(`${tutorialPath\\}`));",
       "\t\t\tconst tutorialCategory = `${3:Your Tutorial Category}`; // replace this with the name of the category for your tutorial",
       "",
       "\t\t\t// then pass name, uri, and category",
```

```suggestion
      "\t\t\t// then pass name, file system path, and category",
```

i think it will be more correct to mention that i tis a file system path. it avoids confusion with the vscode.Uri type.

---

I disagree as this can also be used to pass a true URI, though the example included here doesn't show that. The user has the option to use a http/file url format as well.

---

Might be worth putting a second line (commented out) with an example of using an actual uri-provided file path?

---

Adding this:
```            // or pass an actual https/file uri to use instead
            // const tutorialURL = 'https://my.domain.for.a.didact.file.org/path/to/tutorial.didact.md|adoc'; // replace this with the https/file uri for your tutorial
            // const tutorialUri = vscode.Uri.parse(`${tutorialURL}`);```